rakelib/test.rake

```diff
@@ -8,15 +8,15 @@ namespace "test" do
   end
 
   task "core" do
-    run_rspec(Rake::FileList["spec/**/*_spec.rb"])
+    exit run_rspec(Rake::FileList["spec/**/*_spec.rb"])
```

Comment:
because then is the run_spec the one that is exiting ... it should be the rake task the one thats sets the exit code, not the helper function.